main/guides/zoe/contract-basics.md

```diff
@@ -1,90 +1,25 @@
 # Smart Contract Basics
 
-Before we look at how to make a contract such as the one in [the
-basic dapp](../getting-started/) in the previous section, let's cover some basics.
+This guide is designed to help developers understand how to write smart contracts efficiently and securely. Here, you will find detailed examples that demonstrate different functionalities and use-cases within the context of smart contracts starting from the very basics.
 
-A contract is defined by a JavaScript module that exports a `start` function
-that implements the contract's API.
+## Examples
 
-<<< @/../snippets/zoe/src/01-hello.js#start
+We provide three core examples to illustrate how to implement various smart contract functionalities:
 
-Let's start with a contract with a simple `greet` function:
+### 1. Greetings Contract
+The **Greetings Contract** demonstrates how to create a simple contract that greets the caller. This example is perfect for understanding the basic structure and syntax of smart contracts.
 
-<<< @/../snippets/zoe/src/01-hello.js#greet
+### 2. State Contract
+The **State Contract** shows how to maintain a state or a variable within a smart contract. This is useful for contracts that need to store data or state across transactions.
 
-The `start` function can expose the `greet` function
-as part of the contract API by making it
-a method of the contract's `publicFacet`:
+### 3. Access-Control Contract
+The **Access-Control Contract** provides an example of how to implement access control, ensuring that only authorized users can call certain functions within the smart contract.
```

Comment:
maybe over time we add a 4th example for a basic contract that writes to vstorage to show the difference between storing state like in ```02-state.js```, vs storing in vstorage. Would be a good lesson i think

Reply:
Absolutely - would love to do that.